addon/components/polaris-page.js

```diff
@@ -1,121 +1,107 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import { or } from '@ember/object/computed';
-import layout from '../templates/components/polaris-page';
+import { tagName, layout } from '@ember-decorators/component';
+import template from '../templates/components/polaris-page';
 
 /**
  * Polaris page component.
  * See https://polaris.shopify.com/components/structure/page
  */
-export default Component.extend({
-  classNames: ['Polaris-Page'],
-  classNameBindings: [
-    'fullWidth:Polaris-Page--fullWidth',
-    'singleColumn:Polaris-Page--singleColumn',
-  ],
-
-  layout,
-
+@tagName('')
```

Comment:
Will this break any instances where we have `class` set on `polaris-page` instances? If so, do we have any such instances? Even if we don't, this would still be a breaking change and we should have that in mind when we release this update.

Reply:
Good catch!

Yeah, totally forgot to add backwards support for that and a deprecation. Will add and double-check components already refactored, I know I added this to some, but most likely not all
👍